Testscripts/Linux/TVM-TestCompatibility.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    exit 0
+}
+
+UtilsInit
+homeDir=$(dirname "$0")
+
+if [ ! -e "$homeDir"/tvminfo ]; then
+    LogErr "tvminfo tool is not on the system"
+    SetTestStateAborted
+    exit 0
+fi
+
+chmod +x "$homeDir"/tvminfo
+
+sudo "$homeDir"/tvminfo
+rc=$?
+
+if [ rc == 0]; then
```

Comment:
if `rc` variable is only used once to store exit status, then you can use `$?` directly in if condition.
If there are plans to add code in between these two steps then it should be kept as it is.

Reply:
Will keep this as is for future use cases.